### ISSUE TITLE
Pass multiple subfolders in train validation and test

### ISSUE THREAD

**Issue by drcodex** [Deci-AI/super-gradients#1048]
### 💡 Your Question

For example, let's say, I have multiple image and label folders like the following.
images:
	folder1
	folder2

labels:
	folder1
	folder2 

How do I pass it to dataloader without combining ?

- I'm using coco_detection_yolo_format_train as dataloader

**Comment by Louis-Dupont** [comment]
Hi @drcodex , we currently don't support this type of Dataset out of the box.
If you are working with just a few folders, the most simple workaround would be to unpack these folders into a train/test folders, just make sure to respect the expected structure defined in the documentation https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/training/datasets/detection_datasets/yolo_format_detection.py

You can also implement a new dataset that would support this. You can take all of the existing logic from the `YoloDarknetFormatDetectionDataset` but instead of iterating over the files directly, you would iterate over the folders first.
https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/training/datasets/detection_datasets/yolo_format_detection.py
